includes/Admin/Customizer.php

```diff
@@ -196,7 +196,7 @@ public function register_customizer_settings( WP_Customize_Manager $wp_customize
 		$wp_customize->add_setting(
 			self::STORY_OPTION . '[show_stories]',
 			[
-				'default' => false,
+				'default' => 'false',
```

Comment:
FWIW I think the changes in this file here are not needed.

PHPStan warns about them because the docblock for `add_setting()` is incorrect.

We should instead ignore this error message in the phpstan config